packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -44,7 +44,6 @@ export class CallContext {
   private _state: CallClientState;
   private _emitter: EventEmitter;
   private _atomicId: number;
-  private _batchMode: boolean;
```

Comment:
We will still need batchMode - sometimes we wanna stop stateChanged event for a while in callStatefulClient - especially when we wanna do a bulk update - so needs for 'batch' and state compare are different - I believe we need both